tool/tctl/common/devices.go

```diff
@@ -388,3 +447,34 @@ func findDeviceID(ctx context.Context, devices devicepb.DeviceTrustServiceClient
 
 	return deviceID, assetTag, nil
 }
+
+// currentDeviceCommand marks commands capable of operating against the current
+// device.
+type currentDeviceCommand struct {
```

Comment:
The naming suggests that this itself is a command. What do you think about this instead?

```suggestion
type canOperateOnCurrentDevice struct {
```

I think that looks nice when embedded. In this case, it's saying "oh, the device lock command can operate on the current device"

```
type deviceLockCommand struct {
    canOperateOnCurrentDevice
}
```

Reply:
Better, done.